.travis.yml

```diff
@@ -82,7 +82,7 @@ cache:
 before_install:
   # If you update these versions, make sure you update the versions in the .st2client-install-check
   # and the requirements targets in the Makefile to match
-  - pip install --upgrade "pip>=19.3.1"
+  - pip install --upgrade "pip==20.0.2"
```

Comment:
For the best results it's best to relax patch version by using interval.
This way we still benefit from the important bugfixes and security updates, excluding breaking changes coming in.

Reply:
This is fine, but we need the `Makefile` and the `travis.yml` file to have the exact same version specifier, or else the build "ping pongs" between installing one version in Travis, and then another during the build, and then another in Travis, and then another during the build, which slows the build down.

Reply:
I will revert this to an interval pending the conclusion of StackStorm/discussions#2.